regression/cbmc/gcc_bswap1/main.c

```diff
@@ -0,0 +1,37 @@
+#include <assert.h>
+#include <stdint.h>
+
+#ifndef __GNUC__
+uint16_t __builtin_bswap16(uint16_t);
+uint32_t __builtin_bswap32(uint32_t);
+uint64_t __builtin_bswap64(uint64_t);
+#endif
+
+int main()
+{
+  uint16_t sa = 0xabcd;
+  assert(__builtin_bswap16(sa) == 0xcdab);
+  uint16_t sb;
```

Comment:
Pedantry but, no chance of a nondet_whatever() function call for the things that should be nondet.  Doesn't change behaviour but helps people see what is going on.